app/src/main/assets/map_theme/layers/land.yaml

```diff
@@ -92,6 +92,17 @@ layers:
                     color: global.building_outline_color
                     width: [[15, 1px], [18, 2m]]
 
+        squares:
+            filter: { kind: [pedestrian], $zoom: { min: 17 } }
+            draw:
+                lines:
+                    color: global.path_outline_color
```

Comment:
By the way, if you are going to define a square_color, then you should also define a square_outline_color